sirepo/sim_data/__init__.py

```diff
@@ -484,10 +500,25 @@ def schema(cls):
 
         return cls._memoize(simulation_db.get_schema(cls.sim_type()))
 
+    @classmethod
+    def sim_files_to_run_dir(cls, data, run_dir):
+        for b in cls._sim_file_basenames(data):
+            cls._sim_file_to_run_dir(
+                b.basename,
+                data,
+                run_dir,
+                is_exe=b.get('is_exe', False),
+            )
+
     @classmethod
     def sim_type(cls):
         return cls._memoize(pkinspect.module_basename(cls))
 
+    @classmethod
+    def support_files_to_run_dir(cls, data, run_dir):
+        cls.lib_files_to_run_dir(data, run_dir)
+        cls.sim_files_to_run_dir(data, run_dir)
```

Comment:
A sim_file is a file that lives at user/xxx/type/yyy/<file>. A sim_db_file is any files in the simulation_db (output file, lib file, sim file etc). Maybe I'll change the name to simulation_db_file to make it more clear?

Reply:
I don't like simulation_db, and prefer sim_db. As to the distinction, we need a new name for sirepo/db, since it includes more than simulations. We have srdb.py, which would be an ok name.

For now, we'll live with the subtlety of the naming as you have it.